config.json

```diff
@@ -1344,27 +1344,155 @@
         "dependencyOnly": false
       },
       {
-        "groupId": "androidx.media2",
```

Comment:
We should probably keep the `media2` packages that we are already shipping, as we don't have a way to mark them as deprecated.  We might be responsible for shipping a security update to them or something in the future.